user-interface/frontend/themes/datamanager/components/dialog.css

```diff
@@ -233,9 +233,13 @@ vaadin-dialog-overlay::part(title) {
   min-width: 175px;
 }
 
+.experiment-group-dialog .text-field {
+  min-width: 100px;
+}
+
 .experiment-group-dialog .experimental-group-entry {
   display: grid;
-  grid: auto auto auto / 1fr 175px auto;
+  grid: auto auto auto auto / auto 1fr 175px auto;
   column-gap: 1rem;
   align-items: baseline;
 }
```

Comment:
NIT: The delete button should be at the end of the block not in a new line. However this is NIT since the dialog overhaul is coming soon™️ 
However I think the name field has too much width in comparison to the condition
<img width="1433" alt="Bildschirmfoto 2024-02-14 um 16 24 36" src="https://github.com/qbicsoftware/data-manager-app/assets/29627977/11e73c8f-3050-4e5f-8712-58dc9c702653">